frontstage_api/controllers/case_controller.py

```diff
@@ -126,15 +126,14 @@ def build_full_case_data(case):
 
 def calculate_case_status(case):
     logger.debug('Getting the status of case')
-    case_events = case.get('caseEvents')
+    case_group_status = case.get('caseGroup', {}).get('caseGroupStatus')
     status = None
```

Comment:
We handle the `Not Started` case status a bit strange here. 
We probably don't need to set it to None here or the if statement in the return. Instead we could just have an else statement after the `if... elif`